syntax = "proto3";

package pipeline;

import "pipeline/padding_mode.proto";

message TileFilterConfig
{
  uint32 width = 1;

  uint32 height = 2;

  uint32 stride_x = 3;

  uint32 stride_y = 4;

  PaddingMode padding_mode = 5;
}
